src/GraphBuilder/CdmChangeFeedGraphBuilder.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Threading.Tasks;
+using Akka.Streams;
+using Akka.Streams.Dsl;
+using Arcane.Framework.Services.Base;
+using Arcane.Framework.Sources.CdmChangeFeedSource;
+using Arcane.Stream.Cdm.Extensions;
+using Arcane.Stream.Cdm.Models;
+using Snd.Sdk.Metrics.Base;
+using Snd.Sdk.Storage.Base;
+
+namespace Arcane.Stream.Cdm;
+
+public class CdmChangeFeedGraphBuilder : IStreamGraphBuilder<IStreamContext>
```

Comment:
You can use statically typed graph builder,  like here: https://github.com/SneaksAndData/arcane-stream-blob-storage/blob/1d5e1a42c3ed9522ee281807661f9d3ad05977e6/src/GraphBuilder/BlobStorageGraphBuilder.cs#L54

In this case you don't need to determine type of the stream context dynamically.